src/plugins/mer/meremulatormodedialog.cpp

```diff
@@ -222,9 +220,9 @@ bool MerEmulatorModeDialog::execDialog()
         m_ui->restartEmulatorCheckBox->setEnabled(false);
     }
 
-    if (m_dialog->exec() != QDialog::Accepted) {
+    bool result = (m_dialog->exec() == QDialog::Accepted);
```

Comment:
Possibly without parentheses and with const? :)

Reply:
Parentheses improve readability so I'm not willing to remove them.